### PR TITLE
OSDOCS-9859:updates tp/GA tables in RNs

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):

Issue:
https://issues.redhat.com/browse/OSDOCS-9859

Link to docs preview:
https://76906--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-technology-preview-tables

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -749,13 +749,18 @@ In the following tables, features are marked with the following statuses:
...
 |Technology Preview
-|Technology Preview
+|General Availability
 
 |IPsec external traffic (north-south)
 |Technology Preview
 |General Availability
 |General Availability
 
+|Support for `configure-ovs.sh` Bash script
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:759]: > A MachineConfig object with a customized br-ex bridge

This version might be succint for the table entry?

[the remark this answers is not in the document]
Comment (dfitzmau) [reply release_notes/ocp-4-16-release-notes.adoc:759]: Thanks for this update!

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -749,13 +749,18 @@ In the following tables, features are marked with the following statuses:
 |Admin Network Policy (`AdminNetworkPolicy`)
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
```
Comment (tssurya) [inline release_notes/ocp-4-16-release-notes.adoc:752]: LGTM from ANP side.
@JoeAldinger will do another PR for the paragraph thingy

**Review by tssurya — APPROVED** [review]

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -749,13 +749,18 @@ In the following tables, features are marked with the following statuses:
...
 
 |IPsec external traffic (north-south)
 |Technology Preview
 |General Availability
 |General Availability
 
+|A `MachineConfig` object with a customized `br-ex` bridge
```
Comment (JoeAldinger) [inline release_notes/ocp-4-16-release-notes.adoc:759]: cc @qiowang721 and @cybertron

Comment (qiowang721) [reply release_notes/ocp-4-16-release-notes.adoc:759]: Hi @JoeAldinger Thanks for your helping on the doc update.
I'm thinking about the feature name, the epic name is configure-ovs alternative, but seems it cannot indicate what the feature is explicitly...hmm, how about 'configure the br-ex bridge needed by OVNK using NMState'? @cybertron WDYT?

Comment (qiowang721) [reply release_notes/ocp-4-16-release-notes.adoc]: @JoeAldinger  thanks for updating, LGTM.
Let's wait for comments from @cybertron .

File: release_notes/ocp-4-16-release-notes.adoc (hunk 2 above)
Comment (JoeAldinger) [reply release_notes/ocp-4-16-release-notes.adoc:1209]: done
